src/Tests/dotnet.Tests/CommandTests/ToolUpdateGlobalOrToolPathCommandTests.cs

```diff
@@ -152,6 +152,21 @@ public void GivenAnExistedLowerversionInstallationWhenCallItCanPrintSuccessMessa
                 _packageId, LowerPackageVersion, HigherPackageVersion));
         }
 
+        [Fact]
+        public void GivenAnExistedHigherversionInstallationWhenUpdateToLowerVersionItErrors()
```

Comment:
We should probably have a test for this same scenario succeeding when `--allow-downgrade` is specified.  Likewise, we should have the same two scenarios for local tools.

Reply:
Do you think the `--allow-downgrade` option should be added to tool update (apart from install) as well? Given the inference of the word "update"

Reply:
The `--allow-downgrade` option is added to tool update for this pr